src/Dev/SapphireTest.php

```diff
@@ -594,32 +594,32 @@ public function assertEmailSent($to, $from = null, $subject = null, $content = n
      *
      * @param SS_List|array $matches The patterns to match.  Each pattern is a map of key-value pairs.  You can
      * either pass a single pattern or an array of patterns.
-     * @param SS_List $dataObjectSet The {@link SS_List} to test.
+     * @param SS_List $list The {@link SS_List} to test.
      *
      * Examples
      * --------
      * Check that $members includes an entry with Email = sam@example.com:
-     *      $this->assertDOSContains(array('Email' => '...@example.com'), $members);
+     *      $this->assertListContains(['Email' => '...@example.com'], $members);
      *
      * Check that $members includes entries with Email = sam@example.com and with
      * Email = ingo@example.com:
-     *      $this->assertDOSContains(array(
-     *         array('Email' => '...@example.com'),
-     *         array('Email' => 'i...@example.com'),
-     *      ), $members);
+     *      $this->assertListContains([
+     *         ['Email' => '...@example.com'],
+     *         ['Email' => 'i...@example.com'],
+     *      ], $members);
      */
-    public function assertDOSContains($matches, $dataObjectSet)
+    public static function assertListContains($matches, $list)
```

Comment:
Most phpunit assertions allow you to customise the error message; Shall we add this in while we are adding a new method?.